wikipedia-sentences/flows/query.yml

```diff
@@ -0,0 +1,14 @@
+!Flow
+version: '1'
+with:
+  read_only: true
+  port_expose: $JINA_PORT
+pods:
+  - name: encoder
+    uses: pods/encode.yml
+    timeout_ready: 60000
+    read_only: true
+  - name: indexer
+    uses: pods/index.yml
+    polling: all
+    uses_after: _merge_matches
```

Comment:
no need to `merge_matches` if no shards or parallelization used

Reply:
also `polling`

Reply:
Done!